ajax/libs/vivagraphjs/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "vivagraphjs",
+  "version": "0.8.2",
+  "description": "Graph Drawing Library",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/anvaka/VivaGraphJS.git"
+  },
+  "keywords": [
+    "graph"
```

Comment:
Please add keywords based on [bower.json](https://github.com/anvaka/VivaGraphJS/blob/master/bower.json).

When the keywords are few in `package.json`, first you can see if there is `bower.json` on npm/GitHub and take it as a reference. Second ( if the keywords are still not good enough ), you can try to find out good keywords by googling them, if the GitHub/npm/homepgae displays on the first page of search result, then it's good!